app/scripts/controllers/HomeCtrl.js

```diff
@@ -17,10 +16,15 @@
         
         this.setUsername = function() {
             $cookies.put('blocChatCurrentUser', this.username);
-            this.show = true;
             this.username = null; 
             this.currentUser = $cookies.get('blocChatCurrentUser');
-            console.log(this.currentUser);
+        }
+        
+        this.content = function() {
```

Comment:
I'd suggesting using action/descriptive names for methods like you do above: `setCurrentRoom` and `setUsername`, maybe something like `submitMessage` or `postMessage`.